index.html

```diff
@@ -270,7 +270,7 @@ <h2>
                     documents that consist of the
                     [=relevant global object=]'s [=associated `Document`=]'s
                     [=top-level browsing context=]'s <a data-cite="HTML#nav-document">active document</a>, and
-                    all <a data-cite="HTML#nav-document">active documents</a> in [=nested browsing context=]s of
+                    all <a data-cite="HTML#nav-document">active documents</a> in nested [=browsing context=]s of
```

Comment:
```suggestion
                    all [=navigable/active documents=] in nested [=browsing context=]s of
```

Reply:
I'm going to make that additional change in a separate PR, for easier tracking.

Reply:
Done: https://github.com/w3c/mediacapture-viewport/pull/30